actnow/settings.py

```diff
@@ -64,7 +68,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [Path.joinpath(BASE_DIR, "templates")],
```

Comment:
What's up with this?

Reply:
after defaulting back to the default admin site, we needed to place the login page at the `templates/admin/login.html` folder

Reply:
Cool... but shouldn't enabling [`APP_DIR`](https://docs.djangoproject.com/en/3.2/ref/settings/#app-dirs) be enough?

Reply:
yes, it does but given this `login.html` doesn't belong to the `accounts` app but to the `admin` app, putting it in this level seemed sensible.